Signal/src/contact/OWSContactsManager.m

```diff
@@ -193,17 +209,16 @@ - (void)updateWithContacts:(NSArray<Contact *> *)contacts
 
             [self.avatarCache removeAllImages];
 
-            [self intersectContacts];
-
-            [self buildSignalAccounts];
+            [self intersectContactsWithCompletion:^(NSError *_Nullable error) {
```

Comment:
Previously we were calling buildSignalAccounts immediately, and then again after contacts intersected. 

Previously that made sense, since we had no local store of SignalAccounts. Now that we persist them, there's no reason to rebuild until we have a fresh intersection.

Reply:
Great.